extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/measurementTrackingMachine.js

```diff
@@ -23,6 +23,12 @@ const machineConfiguration = {
       entry: 'clearContext',
       on: {
         TRACK_SERIES: 'promptBeginTracking',
+        SET_TRACKED_SERIES: [
+          {
+            target: 'tracking',
+            actions: ['setTrackedStudyAndMultipleSeries'],
+          },
+        ],
```

Comment:
Are we only adding support for when no tracking is yet occuring?

Worth noting two additional potential transitions that are not yet supported:

- Additive when tracking same study
- Replace when tracking different study
- Additive when "off" (don't track for viewing session state) (restores all as temp measurements?)

Reply:
Good catch, we talked about it but I shall add the first 2 for now. I'm not sure if there is a global setting for "don't track" yet or anything? Wouldn't that be another mode?